tests/Unit/Models/AiWorkflowClaimTest.php

```
@@ -33,6 +36,16 @@ public function testAcquireReturnsFalseWhenDuplicateClaimExists(): void
         $this->assertFalse(AiWorkflowClaim::acquire($database, 10, 'git_pr_match', str_repeat('a', 64)));
     }
 
+    public function testAcquireReturnsTrueWhenStaleInProgressClaimIsReclaimed(): void
+    {
+        $database = $this->createMock(Database::class);
+        $stmt = $this->createMock(\PDOStatement::class);
+        $stmt->method('rowCount')->willReturn(2);
+        $database->method('query')->willReturn($stmt);
+
+        $this->assertTrue(AiWorkflowClaim::acquire($database, 10, 'git_pr_match', str_repeat('a', 64)));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\AiWorkflowClaim' in method 'testAcquireReturnsTrueWhenStaleInProgressClaimIsReclaimed'. (undefined)

(StaticAccess)